packages/lib/file-api-driver-local.ts

```diff
@@ -138,11 +165,10 @@ class FileApiDriverLocal {
 		// });
 	}
 
-	async put(path, content, options = null) {
-		if (!options) options = {};
+	async put(path: string, content: any, options: FileOptions = null) {
 
 		try {
-			if (options.source === 'file') {
+			if (options && 'source' in options && options.source === 'file') {
```

Comment:
please restore the previous behaviour `if (!options) options = {}` that way there's no need to check if option is defined.